translations/fr.json

```diff
@@ -176,16 +171,16 @@
   "tribes.kickoo": "Kickoo",
   "tribes.kickoo.info": "Plages de sable blanc et cocotiers. Abondance de poissons et de fruits. Bienvenue chez les {0}. \n\nLes {0} commencent la partie avec la technologie '{1}' .",
   "tribes.hoodrick": "Hoodrick",
-  "tribes.hoodrick.info": "Les feuilles d'automne jaunes des bois des {0} sont des cachettes parfaites pour ces habitants farcis aux champignons. \n\nLes {0} commencent la partie avec la technologie '{1}'.",
+  "tribes.hoodrick.info": "Les feuilles d'automne jaunes des bois des {0} sont des cachettes parfaites pour ses habitants farcis aux champignons. \n\nLes {0} commencent la partie avec la technologie '{1}'.",
   "tribes.luxidoor": "Luxidoor",
   "tribes.luxidoor.info": "Les {0} aiment les objets chers, les bijoux, les épices rares et les ornements. C'est aussi pour cela qu'ils sont la tribu la plus chère de Polytopia. Leurs citoyens mènent une vie très aisée, drapée dans la plus belle soie violette. Mais survivrons t-ils en dehors des murs de leur capitale bien aimée ? \n\nLes {0} commencent la partie avec une immense capitale fortifiée.",
   "tribes.vengir": "Vengir",
-  "tribes.vengir.info": "Désapprouvés par les autres tribus et poussés en exile dans les terres de cendres. Vont-ils tolérer cette injustice ou déciderons t-il de se venger ? \n\nLes {0} commencent la partie avec la technologie '{1}' et un très puissant Escrimeur.",
+  "tribes.vengir.info": "Désapprouvé par les autres tribus et poussé en exile dans les terres de cendres. Vont-ils tolérer cette injustice ou déciderons t-il de se venger ? \n\nLes {0} commencent la partie avec la technologie '{1}' et un très puissant Escrimeur.",
```

Comment:
The subject in the second sentence is a plurial "vont-ils"... So the past participle have to make the subject agree with the subject.